libraries/chain/transaction_context.cpp

```diff
@@ -114,7 +114,7 @@ namespace eosio { namespace chain {
    {
       trace->scheduled = true;
       is_input = false;
-      apply_context_free = false;
+      //apply_context_free = false; // does CFAs always execute immediately?
```

Comment:
CFA should execute immediately and not at time of delay. slim has been updated and should be correct.